com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/StandardShaderGUI.cs

```diff
@@ -491,13 +492,28 @@ public override void AssignNewShaderToMaterial(Material material, Shader oldShad
                     rimLighting = GetFloatProperty(material, "_UseRimLighting");
                     textureScaleOffset = GetVectorProperty(material, "_TextureScaleOffset");
                 }
+                else if (oldShader.name.Contains("Universal Render Pipeline/Lit"))
```

Comment:
Could you also add support for these URP shaders? 
![image](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/assets/13305729/ad3515c9-479f-4af3-bd7e-71112fadff71)

They likely use similar property names so it might be as easy as checking:
`oldShader.name.Contains("Universal Render Pipeline/Lit") || oldShader.name.Contains("Universal Render Pipeline/Simple Lit") || etc...`

then adding any missing properties.